pan3d/ui/render_area.py

```diff
@@ -0,0 +1,15 @@
+from trame.widgets import html, vuetify
+
+
+class RenderArea(html.Div):
+    def __init__(self, plot_view=None):
```

Comment:
The plot_view won't work

Reply:
Changed in [0c36b45](https://github.com/Kitware/pan3d/pull/28/commits/0c36b45c413e22cc2a0963dde8eb4f875d848413)